main.js

```diff
@@ -99,6 +99,8 @@ ODate.prototype.update = function() {
 		dateString = ODate.format(date, 'date');
 	} else if (format === 'time-ago-limit-4-hours'){
 		dateString = ODate.timeAgo(date, { limit: 4*inSeconds.hour });
+	} else if (format !== '') {
```

Comment:
Ok, I'm unsure how to go about fixing it. There's a lot of different formatting options (https://github.com/Financial-Times/o-date/blob/master/main.js#L17-L35) and to do a check against all of those different combinations would be crazy? Any thoughts?